SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -39,14 +33,34 @@
 #include "Widgets/Notifications/SPopupErrorText.h"
 #include "Widgets/Text/STextBlock.h"
 
-#include "Internationalization/Regex.h"
+#include "SpatialCommandUtils.h"
+#include "SpatialConstants.h"
+#include "SpatialGDKDefaultLaunchConfigGenerator.h"
+#include "SpatialGDKEditorSettings.h"
+#include "SpatialGDKEditorToolbar.h"
+#include "SpatialGDKEditorPackageAssembly.h"
+#include "SpatialGDKEditorSnapshotGenerator.h"
+#include "SpatialGDKServicesConstants.h"
+#include "SpatialGDKServicesModule.h"
+#include "SpatialGDKSettings.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialGDKSimulatedPlayerDeployment);
 
+namespace
+{
+	//Build Configurations
+	const FString Debug(TEXT("Debug"));
```

Comment:
Feel like these could be Spatial constants?

Reply:
These don't really have anything to do with Spatial, so I don't feel like they belong in SpatialConstants.